app/controllers/teams_controller.rb

```diff
@@ -36,15 +28,12 @@ def update
     end
   end
 
-  # DELETE    /teams/:id
   def destroy
-    team = Team.find params[:id]
-    if team
-      team.destroy
+    if @team
+      @team.destroy
       render nothing: true, status: :no_content
-      # render nothing: true, status: :ok
     else
-      render nothing: true, status: :no_found
+      render nothing: true, status: :not_found
     end
```

Comment:
Maybe this will be better IMO:
```ruby
def destroy
  render nothing: true, status: :not_found unless @team

  @team.destroy
  render nothing: true, status: :no_content
end
```